docs.bruno/Testing - Get Data.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing - Get Data
+  type: http
+  seq: 1
+}
+
+get {
+  url: https://api.github.com/users/usebruno
```

Comment:
Consider adding error handling or assertions to check the response of the GET request. This will make the test more robust and meaningful. [important]